ui/components/ui/show-hide-toggle/show-hide-toggle.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useArgs } from '@storybook/client-api';
+import ShowHideToggle from '.';
+
+export default {
+  title: 'Components/UI/ShowHideToggle', // title should follow the folder structure location of the component. Don't use spaces.
+  id: __filename,
+  argTypes: {
+    id: {
+      control: 'text',
+    },
+    ariaLabelHidden: {
+      control: 'text',
+    },
+    ariaLabelShown: {
+      control: 'text',
+    },
+    className: {
+      control: 'text',
+    },
+    dataTestId: {
+      control: 'text',
+    },
+    disabled: {
+      control: 'boolean',
+    },
+    onChange: {
+      action: 'onChange',
+    },
+    shown: {
+      control: 'boolean',
+    },
```

Comment:
Could add title here. I believe the story should show all available props in the controls
```suggestion
    },
    title: {
      control: 'text',
    },
```